tests/system/src/test/java/test/robot/javafx/scene/RobotTest.java

```diff
@@ -490,28 +486,33 @@ public void testMouseClickThrowsNPEForNullArgument() {
     }
 
     @Test
+    @Ignore("Flaky")
     public void testMouseDragPrimary() {
         testMouseDrag(MouseButton.PRIMARY);
     }
 
     @Test
+    @Ignore("Flaky")
```

Comment:
If you want, you can file a follow-up bug to address this, but not necessary.

Reply:
I will be attempting a fix when I look at https://bugs.openjdk.java.net/browse/JDK-8215376

Reply:
Great. In that case, maybe add a comment (or add JDK-8215376 to the `@Ignore` string?)